app/controllers/api/stats_controller.rb

```diff
@@ -26,6 +26,49 @@ def email_verified_signups_per_day
     render(json: fill_in_blank_dates(result).to_json, status: 200)
   end
 
+  # Returns an array of buckets of channels, where each bucket is defined by total channel views
+  def twitch_channels_by_view_count
+    # 0 - 1000 views
+    bucket_one = TwitchChannelDetails.where("stats -> 'view_count' >= ?", "0")
```

Comment:
You can drastically improve the performance of the query by skipping the joins

```
bucket_one = TwitchChannelDetails.select(:channel_id).where("stats -> 'view_count' >= ?", "0").
  where("stats -> 'view_count' < ?", "1000")
```

Additionally, please put the period at the end of the line to make it paste-able into rails console.